networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc

```diff
@@ -18,6 +18,21 @@ OVN configures OVS on each node to implement the declared network configuration.
 OVN-Kubernetes is the default networking solution for {product-title} and {sno} deployments.
 ====
 
+OVN-Kubernetes, which arose from the OVS project, uses many of the same constructs, such as open flow rules, to determine how packets will travel through the network.
+For more information, see the link:https://www.ovn.org/en/[Open Virtual Network website].
+
+OVN-Kubernetes is a series of daemons for OVS that translate virtual network configurations into `OpenFlow` rules.
+`OpenFlow` is a protocol for communicating with network switches and routers, providing a means for remotely controlling the flow of network traffic on a network device, allowing network administrators to configure, manage, and monitor the flow of network traffic.
+
+OVN-Kubernetes provides more of the advanced functionality not available with `OpenFlow`.
+OVN supports distributed virtual routing, distributed logical switches, access control, DHCP and DNS.
+OVN implements distributed virtual routing within logic flows which equate to open flows.
+So for example if you have a pod that sends out a DHCP request on the network, it sends out that broadcast looking for DHCP address there will be a logic flow rule that matches that packet, and it responds giving it a gateway, a DNS server an IP address and so on.
+
+OVN-Kubernetes runs a daemon on each node. There are daemon sets for the databases and for the OVN controller that run on every node.
+The OVN controller programs the OVS switch on the nodes to support the network provider features; egress IPs, firewalls, routers, hybrid networking, IPSEC encryption, IPv6, network policy, network policy logs, hardware offloading and multicast.
```

Comment:
```suggestion
The OVN controller programs the OVS turn on the nodes to support the network provider features; egress IPs, firewalls, routers, hybrid networking, IPSEC encryption, IPv6, network policy, network policy logs, hardware offloading and multicast.
```

Reply:
Vale is flagging switch on as a Red Hat suggested language to use turn on instead.